src/Visualization/CMakeLists.txt

```diff
@@ -53,10 +53,10 @@ else ()
 	target_link_libraries(Visualization Core ${GLFW_LIBRARIES} ${GLEW_LIBRARIES} ${OPENGL_LIBRARIES})
 endif ()
 
-install(TARGETS Visualization
-	RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}/bin
-	LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
-	ARCHIVE DESTINATION ${CMAKE_INSTALL_PREFIX}/lib
-)
-file(GLOB headers "${CMAKE_CURRENT_SOURCE_DIR}/*.h")
-install(FILES ${headers} DESTINATION ${CMAKE_INSTALL_PREFIX}/include)
+# install(TARGETS Visualization
```

Comment:
please remove commented lines. Applied to other cases too.

Reply:
fine with me. that was the "install" script for the c/c++ side of things. it's not working so I decided to hide it entirely. removing the code it's also fine.

Reply:
DONE